package.json

```diff
@@ -193,6 +203,10 @@
     {
       "path": "commonjs/index.js",
       "limit": "212 KB"
+    },
+    {
+      "path": "umd/evergreen.min.js",
```

Comment:
Need to check: are we using the minified commonjs output for size limit checks? If not we may need to figure out why this is so big u minified...

Reply:
No.

Check the build script: `"build-commonjs": "BABEL_ENV=commonjs babel src --out-dir commonjs --ignore '**/stories','**/test' --source-maps inline"`

We didn't use any compress options.

For me I prefer to use: `BABEL_ENV=commonjs babel src --out-dir commonjs --ignore '**/stories','**/test' --minified --no-comments`

Remove source map and comments, then compress it. Same with `build-esm` script.

Then the `commonjs` dir size will reduce from 1.8 MB to 580 KB after I test.

I also test with the `size-limit`. It seems whether I compress it or not, the gzipped sizes are all `210.3 KB`.

Reply:
I think size-limit will automatically apply the gzip. Maybe we should use the unminified version for umd size-limit checks?

Reply:
Even if we compress more, the umd output still can't be reduced to 220kb......because it bundles all dependencies to one file......not like commonjs, only babel transforming.

Reply:
And also, I think we still need to update the `build-commonjs` and `build-esm`. Like I said above: Remove source map and comments, then compress it.

These will reduce the npm package's size.

Reply:
Ah true, I wasn't thinking of that aspect of umd. Ok, I think we can ship this as-is and improve upon it if we need based on actual usage feedback.